pkg/imgpkg/cmd/copy.go

```diff
@@ -242,6 +242,10 @@ func (o *CopyOptions) writeBundleLockOutput(bundle *bundle.Bundle) error {
 func processedImagesMediaType(processedImages *ctlimgset.ProcessedImages) []string {
 	everyMediaType := []string{}
 	for _, image := range processedImages.All() {
+		if image.Image == nil {
```

Comment:
dont we need to recurse into image index as well?

Reply:
We do! we discovered this while TDD'ing. See updated impl.